src/platforms/python/guides/fastapi/index.mdx

```diff
@@ -86,6 +86,37 @@ The parameter `traces_sample_rate` needs to be set when initializing the Sentry
 
 </Note>
 
+## Data collected
```

Comment:
```suggestion
## Data Collected
```

Reply:
changed capitalization of the headline in the last commit